js/src/wcdl/subsection/title/index.scss

```diff
@@ -1,4 +1,5 @@
 .wcdl-subsection-title {
+	position: relative;
```

Comment:
FWIW, I double-checked the code of all our usages of `<Section.Title>`, no other seems to have non-text content, that could be broken by this rule.